### DISCUSSION TITLE
Wazuh API Error 0 after Login

### DISCUSSION THREAD

**Discussion by mrocaruano** [Security-Onion-Solutions/securityonion#4745] [Unsupported Versions] [answered]
So I wanted to check out the Wazuh API, and I logged in by adding a new user using the command ```so-wazuh-user-add user```. When I log in I get the following error ``` {"error":0,"data":{"msg":"Welcome to Wazuh HIDS API","api_version":"v3.13.1","hostname":"securityonion-wazuh-manager","timestamp":"Thu Jul 08 2021 17:25:58 GMT+0000 (UTC)"}}```. Any thoughts on this error, or is the Wazuh API under development?

**Comment by lan94** [comment, marked as answer]
Why do you think this is an error? "Error: 0" means that all is running okay. Maybe you should start with [the documentation](https://documentation.wazuh.com/3.13/user-manual/api/getting-started.html) of the api.

**Reply by mrocaruano** [reply]
Oops, completely missed that part! Thank you so much :+1:
